src/babylon.scene.ts

```diff
@@ -66,7 +66,7 @@ module BABYLON {
     /** Interface defining initialization parameters for Scene class */
     export interface SceneOptions {
         /**
-         * Defines that scene should keep up-to-date a map of geometry to enable fast look-up by Id
+         * Defines that scene should keep up-to-date a map of geometry to enable fast look-up by uniqueId
          * It will improve performance when the number of geometries becomes important.
          */
         useGeometryIdsMap?: boolean;

```

Comment:
I did not change the option parameter name, to preserve backward compatibilty. Should I ?
I updated the comment though.

Reply:
Please change it. It was introduced during current cycle so no problem with breaking changes yet